wiki/Tournaments/TMC/3rd/en.md

```diff
@@ -0,0 +1,599 @@
+2. Teams in the losers bracket must play 2 matches each weekend starting from the Quarterfinals.
+3. Teams that lose a match in the losers' bracket are eliminated from the tournament.
+
+### Win conditions
+
+- In the Qualifiers, teams need to place in the top 32 seeded teams in order to advance to the double-elimination stage.
+- In the Round of 32 and Round of 16, teams need to win 4 maps to win a match (best of 7).
+- In the Quarterfinals and Semifinals, teams need to win 5 maps to win a match (best of 9).
+- In the Finals, teams need to win 6 maps to win the match (best of 11).
+- In the Grand Finals, teams need to win 7 maps to win the match (best of 13).
+
+### Match instructions
+
+1. A referee will create a multiplayer room 15 minutes before the scheduled match time. Players must join the lobby in that period.
+   - Team captains will be notified of their match via ping on Discord by the referee of their match 15 minutes before the match is scheduled to commence.
+   - The room settings are: Game mode: "osu!mania"; Team mode: "Team Vs"; and Score mode: "ScoreV2". The room name must follow this pattern: `THMC3 ({Stage}): ({Blue Team}) vs ({Red Team})`. The team mentioned first in the room name must be the blue team, the team mentioned second in the room name must be the red team.
+2. Each captain can ban **one beatmap** from the pool. These beatmaps can not be picked by any team for the entire duration of the match.
+3. Beatmap selection will alternate between each captain selecting a beatmap out of the mappool.
+4. Each captain must use `!roll` once in `#multiplayer`.
+   - The winner of the `!roll` will select which team will pick first.
+   - The loser of the `!roll` will select which team will ban first.
+5. Teams will have 2 minutes to pick a beatmap and 2 minutes to get ready. If a team takes more time than allotted, the procedure adopted will be as follows:
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee
+   - On subsequent occurrences:
+     - For a pick timer: a random map will be chosen from the mappool using `!roll X`, where X is the number of maps that were neither picked nor banned, excluding the Tiebreaker.
+     - For a ready timer: the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
+   - Repeat offenders may receive further sanctions from the Tournament Management.
+   - Each team will receive one "tactical timeout" of two minutes, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional*.
```

Comment:
when bolding, italicising or linking a whole sentence, the terminating punctuation goes on the inside

```suggestion
   - Each team will receive one "tactical timeout" of two minutes, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional.*
```